server/testdriver/core/src/main/java/org/infinispan/server/test/core/HotRod.java

```diff
@@ -0,0 +1,48 @@
+package org.infinispan.server.test.core;
+
+import org.infinispan.client.hotrod.RemoteCache;
+import org.infinispan.client.hotrod.RemoteCacheManager;
+import org.infinispan.client.hotrod.configuration.ConfigurationBuilder;
+import org.infinispan.configuration.cache.CacheMode;
+
+/**
+ * @since 10
+ */
+public class HotRod extends Base<HotRod> {
+   private final TestServer testServer;
+   private final TestClient testClient;
+   ConfigurationBuilder clientConfiguration = new ConfigurationBuilder();
+
+   public HotRod(TestServer testServer, TestClient testClient) {
+      this.testServer = testServer;
+      this.testClient = testClient;
+   }
+
+   public HotRod withClientConfiguration(ConfigurationBuilder clientConfiguration) {
+      this.clientConfiguration = clientConfiguration;
+      return this;
+   }
+
+   public <K, V> RemoteCache<K, V> get() {
+      RemoteCacheManager remoteCacheManager = testClient.registerResource(testServer.newHotRodClient(clientConfiguration));
+      String name = testClient.getMethodName(qualifier);
+      return remoteCacheManager.getCache(name);
+   }
+
+   public <K, V> RemoteCache<K, V> create() {
```

Comment:
`create` also seems ambiguous to me, it doesn't say it's creating the cache on the server, so it could be interpreted as "create the `RemoteCache` instance so that `get()` can use it".

Reply:
again, all this already existed